README.md

```diff
@@ -164,28 +164,14 @@ described above.
 ### Local usage
 
 You can use the framework to run tests on a local system without a scheduler. After spack has
-been installed following the instructions given above, it needs to be told about the software
-installed on the system. On supported HPC systems this is handled by the spack environment.
+been installed following the instructions given above, it needs to be aware of the software
+installed on the system. On supported HPC systems this is pre-set in the spack environment.
 By default an empty spack environment will be created in `spack-environments/generic/default`
-when you run ReFrame for the first time. To find locally installed compilers run
-`spack compiler find`. To find locally installed packages run `spack external find`. This
-does not necessarily find everything and some packages may have to be added manually,
-in particular it is worth making sure that a local MPI installation is added to the environment
-to avoid lengthy recompilation of the MPI library. If no MPI package is found (found packages
-are added to the `packages.yaml` of your local spack cache), add
-```yaml
-  packages:
-    <your installed MPI implementation, e.g. openmpi>:
-      externals:
-      - spec: <your installed MPI implementation@version, e.g. openmpi@4.1.2>
-        prefix: <path to MPI installation, e.g. /usr>
-```
-To either `spack-environments/generic/default/spack.yaml` or `packages.yaml`. The name,
-version, and prefix may vary depending on your local operating system. The examples given above
-are the defaults on Ubuntu 22.04.
```

Comment:
Everything above beginning with "This does not necessarily find everything..." should be either here or in the `CONTRIBUTING.md`. Probably the latter.

Reply:
I've removed that from here and refer to `CONTRIBUTING.md`